Appium/Dockerfile

```diff
@@ -76,14 +76,15 @@ RUN mkdir -p ~/.android && \
     touch ~/.android/repositories.cfg && \
     echo y | sdkmanager "platform-tools" && \
     echo y | sdkmanager "build-tools;$ANDROID_BUILD_TOOLS_VERSION" && \
-    echo y | sdkmanager "platforms;$ANDROID_PLATFORM_VERSION"
+    echo y | sdkmanager "platforms;$ANDROID_PLATFORM_VERSION" && \
+    echo y | sdkmanager --update
 
 ENV PATH=$PATH:$ANDROID_HOME/platform-tools:$ANDROID_HOME/build-tools
 
 #====================================
 # Install latest nodejs, npm, appium
 #====================================
-ARG APPIUM_VERSION=1.7.0-beta
+ARG APPIUM_VERSION=1.7.2
```

Comment:
It contains only the default value. we update the version by passing the environment variable to travis, not by hard coded things. so this line is not needed to be changed. you can read about it here: https://docs.docker.com/engine/reference/builder/#environment-replacement 

It means the latest docker image contains already version 1.7.2 
<img width="928" alt="screen shot 2018-03-08 at 20 52 09" src="https://user-images.githubusercontent.com/22218312/37172976-af88062a-2312-11e8-8008-ebdcf61618ea.png">